webroot/js/cake-core.js

```diff
@@ -0,0 +1,101 @@
+var postLinkFunction = function () {
```

Comment:
@ankr Yeah, you are right. I will change that.
@ionas Sorry, but I don't really get your point. The functionality with this script and stylesheet is already implemented in CakPHP 3. These are changes to support CSP (Content Security Policy): Separating inline js and css in separate files. So this is the same functionality but in separated loaded files.